### ISSUE TITLE
why we should use three weight files?

### ISSUE THREAD

**Issue by jackyysu** [oist/Usiigaci#6]
Thank you for your perfect work!
I download h5 files from netdisk you shared, it works, but I don't understant why we got three weight files after training. 
thank you.

best regards,

**Comment by hftsai** [comment]
We demonstrated that a small training dataset is enough to give acceptable results by retraining on a pretrained model weight. However, the accuracy is absolutely not perfect. Also, currently there is this "flickering" false negative segmentation from time to time by CNN, in which a target will suddenly not recognized in one frame by one model weight. However, another model weight that's slightly different (obtained from another epoch by training with the exactly the same training data) may recognize it. 
Thus, to get move accurate results and avoid flickering, we can combined the results from inference with multiple model weights. 

but of course, you can just inference with just one modelweight.
this can be modified in inference.py line 296, by specifying only one model.

In the training process, currently, we saved all the modelweights in every epoch. 
We are adding automatically saving of best modelweights after each training.

**Comment by jackyysu** [comment]
Thank you for your kindly reply, I have got the idea.
